src/hotspot/cpu/x86/x86_32.ad

```diff
@@ -1088,18 +1059,19 @@ uint MachSpillCopyNode::implementation( CodeBuffer *cbuf, PhaseRegAlloc *ra_, bo
       // mem -> mem
       int src_offset = ra_->reg2offset(src_first);
       int dst_offset = ra_->reg2offset(dst_first);
-      return vec_stack_to_stack_helper(cbuf, do_size, src_offset, dst_offset, ireg, st);
+      vec_stack_to_stack_helper(cbuf, src_offset, dst_offset, ireg, st);
```

Comment:
None of these functions (vec_stack_to_stack_helper, vec_mov_helper, vec_spill_helper, vec_spill_helper) returns a value that is used on either x86 or x64 now. 

Please make them all return void.

Reply:
Thanks @neliasso for your review.
Will do it later.

And I'll close this pr and create a new one into jdk16 since this issue had been targeted to jdk16.